test-collateral/Makefile

```diff
@@ -128,9 +128,10 @@ css-$(TEE).bin: $(RUNTIME_MANAGER_CSS_BIN)
 get_random_policy.json: pgen pgen css-$(TEE).bin client_rsa_cert.pem random-source.wasm
 	./pgen --certificate client_rsa_cert.pem --capability "output : $(READ_RIGHT), random-source.wasm : $(WRITE_RIGHT)" \
 		--binary random-source.wasm --capability "output : $(WRITE_RIGHT)" \
-       --veracruz-server-ip 127.0.0.1:3011 --proxy-attestation-server-ip 127.0.0.1:3010 --output-policy-file $@ --certificate-expiry $(CERTIFICATE_EXPIRY) \
+		--veracruz-server-ip 127.0.0.1:3011 --proxy-attestation-server-ip 127.0.0.1:3010 --output-policy-file $@ --certificate-expiry $(CERTIFICATE_EXPIRY) \
```

Comment:
It's not the fault of this pull request, but this `Makefile` is very repetitive and hard to read.

Reply:
Could we factorise those long calls into a call to a function? The only variable seems to be the capability and the binary name.
Example:
```
define call_pgen
    ./pgen --certificate client_rsa_cert.pem --capability "$(1)" --binary "$(2)" <...>
endef

get_random_policy.json: pgen pgen css-$(TEE).bin client_rsa_cert.pem random-source.wasm
    $(call call_pgen, "output : $(READ_RIGHT), random-source.wasm : $(WRITE_RIGHT)", "random-source.wasm")
```

Incidentally, `--enclave-debug-mode` is not always set, is that on purpose?

Reply:
I'm going to mark this as resolved. It would be nice to simplify this `Makefile`, but that would be a different issue.

Reply:
@egrimley-arm, I agree that the Makefile needs simplifying, but I think that is outside of the scope of this changeset and should be handled in another PR.